modules/serverless-rn-1-25-0.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies
+//
+// * serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-25-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.25.0
+
+{ServerlessProductName} 1.25.0 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1.25.0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.4.
+* {ServerlessProductName} now uses Knative Eventing 1.4.
+* {ServerlessProductName} now uses Kourier 1.4.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.4.
+* {ServerlessProductName} now uses Knative Kafka 1.4.
+* The `kn func` CLI plug-in now uses `func` 1.7.0.
+
+* {FunctionsProductName} is now generally available (GA). Only the Quarkus runtime is generally available. The other provided runtimes are available as a Technology Preview (TP). See xref:../serverless/functions/serverless-functions-getting-started.adoc#serverless-functions-getting-started[Getting started with functions].
+* Integrated development environments (IDE) plug-ins for creating and deploying functions are now available for link:https://github.com/redhat-developer/vscode-knative[Visual Studio Code] and link:https://github.com/redhat-developer/intellij-knative[IntelliJ].
+* Knative Kafka Broker is now generally available (GA). Knative Kafka Broker is a highly performant implementation of the Knative Broker API, directly targeting Apache Kafka.
++
+It is recommended to not use the MT-Channel-Broker, but the Knative Kafka Broker instead.
+* Knative Kafka Sink is now generally available (GA). A `KafkaSink` takes a `CloudEvent` and sends it to an Apache Kafka Topic. Events can be specified in either Structured or Binary mode.
+
+* Enabling TLS for internal traffic is now available as a Technology Preview. See the "Global configuration" section (FIXME link once https://github.com/openshift/openshift-docs/pull/50404 is merged) for details.
+// the link is https://50404--docspreview.netlify.app/openshift-enterprise/latest/serverless/security/serverless-config-tls.html
+
+[id="fixed-issues-1.25.0_{context}"]
+== Fixed issues
+
+* Previously, Knative Serving had an issue where the readiness probe failed if the container was restarted after a liveness probe fail. This issue has been fixed.
+
+[id="known-issues-1.25.0_{context}"]
+== Known issues
+
+* The Federal Information Processing Standards (FIPS) mode is disabled for Kafka broker, Kafka source, and Kafka sink.
```

Comment:
Kafka broker, Kafka source, and Kafka sink are capitalized in section above - pointing out for consistency